### PR TITLE
Extend query builder templates for ES 7.10

### PR DESCRIPTION
**Pull request by rahulPalliyalil**
### What

This PR extends query builder templates for ES 7.10. This either reuses or makes slight fixes to the existing templates against ES 2.2

### How to review

Please find the trello ticket: https://trello.com/c/AZZovMI4/953-extend-query-builder-templates-for-es-710

Please Note: Most of the template combination are tested locally against ES 7.10

### Who can review

Anyone except me

### CODE REVIEW

File: templates/search/v710/weightedQuery.tmpl [hunk 1]
```diff
@@ -0,0 +1,48 @@
+"function_score": {
+"query": {  {{template "coreQuery.tmpl" .}} },
+
+      "functions": [
+        {
+          "filter": {
+            "term": {
+              "_type": "bulletin"
```
Comment (nshumoogum) [inline templates/search/v710/weightedQuery.tmpl:8]: Should these be `type` not `_type`?

File: templates/search/v710/contentFilterPublished.tmpl [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "bool": {
+    "should":[
+          {
+            "bool": {
+              "must_not": {
+                "term": {
+                  "description.cancelled": true
```
Comment (nshumoogum) [inline templates/search/v710/contentFilterPublished.tmpl:8]: Is it worth updating this? Remove description object as model has been flattened

**Review by nshumoogum — APPROVED** [review]

File: templates/search/v710/weightedQuery.tmpl (hunk 1 above)
Comment (rahulPalliyalil) [reply templates/search/v710/weightedQuery.tmpl]: good spot, fixed them.,

File: templates/search/v710/contentFilterPublished.tmpl (hunk 2 above)
Comment (rahulPalliyalil) [reply templates/search/v710/contentFilterPublished.tmpl]: fixed them.

**Review by nshumoogum — APPROVED** [review]

**Pull request merged**